### ISSUE TITLE
System.Runtime.InteropServices.RuntimeInformation version issue

### ISSUE THREAD

**Issue by the800kid** [stripe/stripe-dotnet#2220]
My C# .NET class library targets .NET Framework 4.6.1.  I have installed the latest Stripe version 39.13.0 through Nuget package manager in Visual Studio 2019.  When I attempt to compile my project I am getting "Could not locate the assembly "System.Runtime.InteropServices.RuntimeInformation, Version=4.0.2.0".  When I installed Stripe, it did add the reference to this library.  I looked in the NuGet package manager, and the latest version "4.3" of the library was installed.  This is actually version 4.0.1.0 when you look at the properties of the reference in Visual Studio.  

In order to get my project to compile, I needed to reference "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Facades\System.Runtime.InteropServices.RuntimeInformation.dll" and updated the redirect in my app.config to 4.0.2.0.  I did both before installing the Stripe library.  At this point my project is now compiling.

**Comment by ob-stripe** [comment]
Hi @the800kid, sorry for not replying sooner!

Can you post the exact snippet from your `app.config` file regarding the RuntimeInformation dependency? I'm not familiar with this issue, but I found [this StackOverflow thread](https://stackoverflow.com/a/49078929/5307473) that suggests adding this:
```xml
<dependentAssembly>
  <assemblyIdentity name="System.Runtime.InteropServices" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-4.1.1.0" newVersion="4.3.0" />
</dependentAssembly>
```

Can you give it a try and post the results here?

**Comment by the800kid** [comment]
Hi Olivier,

No worries.  I already got it working with the changes I mentioned in my original post.  The suggested addition was not needed.

However, I did the following in an attempt to reproduce my original issue:

- Removed the manual reference I added to  "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework.NETFramework\v4.7.2\Facades\System.Runtime.InteropServices.RuntimeInformation.dll".
- Uninstalled Stripe package
- Uninstalled System.Runtime.InteropServices.RuntimeInformation packages.
- Removed the dependency entry in app.config for System.Runtime.InteropServices.RuntimeInformation.
- Installed Stripe 39.24.0 (I tried 39.13.0 as well).  This also installed System.Runtime.InteropServices.RuntimeInformation 4.3.

App.config entry created by new package install
`      <dependentAssembly>
        <assemblyIdentity name="System.Runtime.InteropServices.RuntimeInformation" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
        <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
      </dependentAssembly>
`

I have two projects that I did this with and both compile with no issue.  Not sure what the issue was, but it must have been on my end and worked itself out?  Thank you for looking into this, and sorry that it seems I wasted your time.  At the time I upgraded to the newest Stripe version, and my project stopped compiling.
